### PR TITLE
Update twine-cert.txt

### PR DESCRIPTION
**Pull request by tonishtereva**
Twine Keys for 1.34

### CODE REVIEW

File: twine-cert.txt [hunk 1]
```diff
@@ -3256,3 +3256,236 @@
...
+[infschg_detail_page_no_mask_mandatory_subtitle]
+    de = Befreit bis %d
+    en = Exempt until %d
+
+[infschg_detail_page_no_mask_mandatory_federal_state]
+    de = In %d
+    en = In %d
+
+[infschg_detail_page_no_mask_mandatory_copy_1]
+    de = Sie sind in diesem Bundesland  in einigen Situationen von der Maskenpflicht ausgenommen; eine Maske ist dennoch empfohlen.
```
Comment (timokoenig) [inline twine-cert.txt:3336]:
```suggestion
    de = Sie sind in diesem Bundesland in einigen Situationen von der Maskenpflicht ausgenommen; eine Maske ist dennoch empfohlen.
```

File: twine-cert.txt [hunk 2]
```diff
@@ -3256,3 +3256,236 @@
...
+[infschg_popup_choose_federal_state_button]
+    ref = OK
+    
+[infschg_start_screen_status_green]
+    de = Keine Maskenpflicht
+    en = No mask required
+    
+[infschg_start_screen_status_blue]
+    de = Maskenpflicht
+    en = Mask required
```
Comment (timokoenig) [inline twine-cert.txt:3289]: In general it would be more like a `mask mandate`

File: twine-cert.txt [hunk 3]
```diff
@@ -3256,3 +3256,236 @@
...
+[infschg_detail_page_mask_status_uncertain_button]
+    de = Bundesland ändern
+    en = Change federal state
+
+[infschg_detail_page_no_valid_certificate_title]
+    de = Kein gültiges Zertifikat
+    en = No valid certificate
+
+[infschg_detail_page_no_valid_certificate_copy]
+    de = In ihrem Bundesland können Sie bis zu 24h von der Maskenpflicht befreit werden. Beachten Sie, dass in Ihrem Bundesland andere Bestimmungen gelten können. Denken Sie auch daran, dass die Befreiung nicht für Orte gilt, an denen eine besondere Maskenpflicht gilt: z.B. Arztpraxen, Krankenhäuser oder dem Personenfernverkehr.
```
Comment (timokoenig) [inline twine-cert.txt:3420]:
```suggestion
    de = In Ihrem Bundesland können Sie bis zu 24h von der Maskenpflicht befreit werden. Beachten Sie, dass in Ihrem Bundesland andere Bestimmungen gelten können. Denken Sie auch daran, dass die Befreiung nicht für Orte gilt, an denen eine besondere Maskenpflicht gilt: z.B. Arztpraxen, Krankenhäuser oder dem Personenfernverkehr.
```

**Review by timokoenig — CHANGES_REQUESTED** [review]

File: twine-cert.txt (hunk 2 above)
Comment (Ein-Tim) [reply twine-cert.txt:3289]: "Mask mandatory"

**Review by timokoenig — APPROVED** [review]

**Pull request merged**